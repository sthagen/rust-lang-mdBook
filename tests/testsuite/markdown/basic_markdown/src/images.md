# Images

![Image *"alt"* & \"
    `"text" & <stuff>`
    [url](/foo)
    <em>html</em>
    ---
    hard\
    break
](https://rust-lang.org/logos/rust-logo-256x256.png)

![Image with title](https://rust-lang.org/logos/rust-logo-256x256.png "Some title")
