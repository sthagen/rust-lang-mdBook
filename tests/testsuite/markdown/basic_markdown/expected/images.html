<h1 id="images"><a class="header" href="#images">Images</a></h1>
<p><img src="https://rust-lang.org/logos/rust-logo-256x256.png" alt="Image “alt” &amp; &quot; &quot;text&quot; &amp; &lt;stuff&gt; url &lt;em&gt;html&lt;/em&gt; — hard break "></p>
<p><img src="https://rust-lang.org/logos/rust-logo-256x256.png" title="Some title" alt="Image with title"></p>